webapp/src/container.tsx

```diff
@@ -71,6 +73,16 @@ export class SideDocs extends data.Component<ISettingsProps, {}> {
         this.firstLoad = false;
     }
 
+    private handleEscape = (e: KeyboardEvent) => {
+        let charCode = (typeof e.which == "number") ? e.which : e.keyCode
```

Comment:
I don't think we should be doing this, there's an assumption of where the focus is for this to work. When you first focus is switched to inside the side docs, this doesn't work. The side docs is not like a modal, doesn't need an escape to get out of it. and in the case of the modal, usually when you escape you get put back to where you were last (which isn't what you're doing in this case).